pipeline/test/pipeline.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("pipeline", () => {
+  it("exists", () => {
+    expect(2 + 2).toBe(4);
```

Comment:
```
npm install peano-axioms
```

Reply:
previously I did not manage to refrain from putting this terrible joke into code https://github.com/wellcomecollection/concepts-pipeline/blob/0ea38f4060058678b0d1c52ef0473d9839e97415/ingestor/src/test/scala/weco/concepts/ingestor/Main.scala#L6-L10